.circleci/config.yml

```diff
@@ -50,6 +50,12 @@ jobs:
             . env/bin/activate
             make -C docs/ doctest
 
+      - run:
+          name: linkcheck
+          command: |
+            . env/bin/activate
+            make -C docs/ linkcheck linkcheck_retries=3 linkcheck_anchors=False linkcheck_ignore=[https:\/\/codecov.*]
```

Comment:
Are you sure any of these options (`linkcheck_retries`, etc) actually propagate to `sphinx-build`?

Reply:
No. I put them here as a reminder of what might need to be set in other repos but when I tried setting more than a single value for `linkcheck_ignore` I could not get it to work properly. I did not finish work on that yet. Here I put this for PR to get feedback on whether we want to add this testing before I spend much time getting it to work

Reply:
Oh, I think we should definitely add some link checking. Maybe even make it a periodic job. But finishing this PR would be a great start.